index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Hello, Welome!</title>
+<link rel="stylesheet" href="style.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+
+<body>
+   <navbar class="navbar-fixed-top">
+<p>Welcome! <img src="/home/edwin/Portfolio-webpage/Images/Capacity.jpg"> <img src="/home/edwin/Portfolio-webpage/Images/Icon - Menu.png"> <img src="/home/edwin/Portfolio-webpage/Images/Wifi.png"> </p>
+   </navbar>
```

Comment:
- please, stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=VJYlSbMqq9yrCYDy-0), here are the hamburger buttons needed for mobile, and remove other parts..👌👍